arrow/translate.py

```diff
@@ -52,10 +52,10 @@ def _b64_decode(encoded_value):
 def _list_enums(schema):
     object_field = next(f for f in schema['fields'] if f['name'] == 'object')
     types = [t for t in object_field['type'] if t['name'] != 'Metadata']
-    enums = [(entity_type['name'], field['name'])
+    enums = {(entity_type['name'], field['name'])
```

Comment:
nice catch ! Since enums are meant to have unique values in general, {set} seems like a good choice overall esp. if theres a performance boost (+1 for supporting Q4 goals)

Reply:
Yes. My profiling/performance testing showed a lot of time being spent on [line 33](https://github.com/DataBiosphere/arrow/pull/3/files#diff-85dd3e64aca81f91157c0d1d6fcf6ef3L33), specifically `(entity_type, key) in enums`. The test case was an import of 1120 cases which was taking about 30+ seconds when deployed to GAE. `in` on a set is so much faster than on a list that this one changed alone shaved off ~10 seconds!